includes/wikia/tasks/Tasks/RenameUserPagesTask.class.php

```diff
@@ -0,0 +1,64 @@
+<?php
+namespace Wikia\Tasks\Tasks;
+
+use Title;
+use User;
+use Wikia;
+
+class RenameUserPagesTask extends BaseTask {
+	const NAMESPACES = [
+		2, // User
+		3, // User talk
+		500, // User blog
+		1200 // Message Wall
+	];
+
+	/**
+	 * Get wikis where this user has an user page, user blog, Message Wall etc.
+	 * @param string $oldUserName
+	 * @return int[] array of wiki IDs
+	 */
+	public static function getTargetCommunities( string $oldUserName ) {
+		global $wgExternalDatawareDB;
+
+		$dbr = wfGetDB( DB_SLAVE, [], $wgExternalDatawareDB );
+
+		return $dbr->selectFieldValues( 'pages', 'page_wikia_id', [
+			'page_namespace' => static::NAMESPACES,
+			'page_title' => $oldUserName
```

Comment:
Should we change it to `LIKE` condition and move user page sub-pages as well?

Reply:
Oh, wait, it's covered in `renameLocalPages`

Reply:
There shouldn't be a need to do that:
* if Wall root page does not exist, Wall is broken
* if blog listing page does not exist, Blogs are also broken
* if user does not have user page, user talk page, Message Wall, or blog, but somehow has user or user talk subpages - do we want to check for such situation?